pages/vi/profiles/FazeelAyaz.md

```diff
@@ -0,0 +1,21 @@
+# Fazeel Ayaz
+
+![image](https://github.com/user-attachments/assets/c0d400d8-218a-4b9f-905f-09dad68aea22)
+
+
+### Location: Dallas, Texas 🏴
+### Time Zone: CST
+### OS: Windows 10 (Ver 22H2) 💻
+
+### My name is Fazeel and I will be graduated from UT Dallas this December with a Bachelor's in Computer Engineering. My hobbies are playing online FPS and I like watching documentaries about history and nature. 
```

Comment:
Looks good overall! 🎉

Just wanted to make sure you are aware that Markdown is meant to be simple and typically we are unable to custom the text other than applying bold/italic.

Using Markdown header `###` to make text larger doesn't align with their intended semantic meaning in Markdown. Instead, you'd typically use `#` for larger headings and more `#` symbols for smaller headings, which reflects a clear content hierarchy.